Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -551,7 +551,7 @@
   <bean id="idpResourceFactory" class="org.pentaho.platform.spring.security.saml.resources.MetadataResourceFactory">
     <argument>
       <map key-type="java.lang.String" value-type="java.lang.String">
-        <entry key="org.opensaml.util.resource.HttpResource" value="${saml.idp.metadata.url}" />
+        <entry key="org.pentaho.platform.spring.security.saml.resources.PentahoHttpResource" value="${saml.sp.metadata.url}" />
```

Comment:
I believe the URL might be incorrect here with a copy-paste issue: saml.sp.metadata.url vs saml.idp.metadata.url

Reply:
You are right ! I fixed it in a commit
Kind regards 
Detlef